### PR TITLE
Feat: improve console output

### LINKED ISSUE #186: Unify console prints
Console prints are done everywhere in the codebase, we should now unify the way it is done, and consider using the features of `rich` to make a clean output.
We can use the `getLogger` mechanism to unify this.

### PR DESCRIPTION
**Pull request by Bernardstanislas**
Closes #186

### CODE REVIEW

File: src/prompt.py [hunk 1]
```diff
@@ -1,35 +1,42 @@
...
-init(autoreset=True)
+from rich import print as rich_print
+from src.console import console
 
 
 def on_watch_start(exercise_path):
-    print(f"{Style.BRIGHT}🤖🤖🤖 Watch mode started.")
-    print(f"You can start to work on exercise {exercise_path}.\n")
+    rich_print("[bold]:robot::robot::robot: Watch mode started.[/bold]")
+    rich_print(f"You can start to work on exercise {exercise_path}.\n")
```
Comment (tekkac) [inline src/prompt.py:7]: Could be useful to specify colors variable in a separate file.
We could later use different color schemes for core curriculum vs diploma awarding tracks.

**Review by tekkac — APPROVED** [review]
image
Any way to suppress control keys from the input? This happened when I tried to scroll.

File: src/prompt.py (hunk 1 above)
Comment (Bernardstanislas) [reply src/prompt.py:7]: sure, this is a `good enough` solution for the current need, but we will later bring some print hierarchy with color schemes

**Comment by Bernardstanislas** [comment]
I couldn't fin a way to suppress these :/ maybe it is written somewhere in the rich documentation but I could not find it

**Pull request merged**